packages/api/src/external/aws/address.ts

```diff
@@ -0,0 +1,38 @@
+import { Address } from "../../domain/medical/address";
+import {
+  getCoordinatesFromLocation,
+  makeLocationClient,
+} from "@metriport/core/external/aws/location";
+import { Config } from "../../shared/config";
+import { Coordinates } from "@metriport/core/external/aws/location";
+
+/**
+ * Geocodes a list of addresses using Amazon Location Services.
+ * @param addresses
+ * @returns
+ */
+export async function geocodeAddresses(addresses: Address[]): Promise<Coordinates[]> {
+  const indexName = Config.getPlaceIndexName();
+  const awsRegion = Config.getAWSRegion();
+  const client = makeLocationClient(awsRegion);
+
+  const coords = [];
```

Comment:
We should not create objects without type.

nit: could even convert the `for` to a `.map()` + `Promise.all()` and then we don't have to declare an array, with the added benefit of running the search in parallel. But NABD this last paragraph.

Reply:
Yea, this whole function will be changing soon with this follow-up issue - [#1326](https://github.com/metriport/metriport-internal/issues/1326)